appcenter-auth/android/src/main/java/com/microsoft/appcenter/reactnative/auth/AppCenterReactNativeAuthModule.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter.reactnative.auth;
+
+import android.app.Application;
+
+import com.facebook.react.bridge.BaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+import com.microsoft.appcenter.AppCenter;
+import com.microsoft.appcenter.auth.Auth;
+import com.microsoft.appcenter.reactnative.shared.AppCenterReactNativeShared;
+
+public class AppCenterReactNativeAuthModule extends BaseJavaModule {
+
+    public AppCenterReactNativeAuthModule(Application application) {
+        AppCenterReactNativeShared.configureAppCenter(application);
+        if (AppCenter.isConfigured()) {
+            AppCenter.start(Auth.class);
```

Comment:
From https://docs.microsoft.com/en-us/appcenter/sdk/auth/android

> The Auth module must be started at the same time as other modules, in a AppCenter.start call. If you decide to start the Auth service separately using AppCenter.startService, the app will not sign in automatically after the restart and the user will have to call signIn again.

We might need to start Auth from react-native shared (in configureAppCenter) using reflection or think about another solution.

Does not have to be fixed for this particular bootstrap PR but please add a work item to track that issue.
It's the same for iOS.

Reply:
Sounds good